README.md

```diff
@@ -8,7 +8,9 @@ Documentation on Corda Enterprise Network Manager (CENM) can be found at [CENM D
 | ------------- |---------------------|
 | 1.2           | git checkout v1.2   |
 | 1.3           | git checkout v1.3   |
```

Comment:
I think we need to remove rows with 1.3 and 1.4. We keep only the latest of each release.

Reply:
Agreed